bundles/ch.elexis.stickynotes/src/ch/elexis/stickynotes/ui/StickyNotesView.java

```diff
@@ -134,19 +134,20 @@ private void deselect() {
 	private void doSelect(Patient pat) {
 		if (pat == null) {
 			deselect();
-
 		} else {
-
 			actPatient = pat;
 			actNote = StickyNote.load(actPatient);
 			etf.setText(actNote.getText());
-			// form.setText(actPatient.getLabel());
 			setPartName(Messages.StickyNotesView_StickyNotesNameDash + actPatient.getLabel());
-			RGB rgb = PreferenceConverter.getColor(prefs, Preferences.COLBACKGROUND);
-			UiDesk.getColorRegistry().put(Preferences.COLBACKGROUND, rgb);
+			RGB rgbBackground = PreferenceConverter.getColor(prefs, Preferences.COLBACKGROUND);
+			RGB rgbForeground = PreferenceConverter.getColor(prefs, Preferences.COLFOREGROUND);
+			if (rgbBackground.equals(rgbForeground)) {
+				rgbBackground = new RGB(255, 255, 0);
+				rgbForeground = new RGB(0, 0, 0);
+			}
```

Comment:
You are hardcoding the values to black and white if they are equal. You could also calculate the inverse color of the background? Not necessary, but just a suggestion :)